packages/beacon-node/src/network/reqresp/handlers/lightclientUpdate.ts

```diff
@@ -0,0 +1,9 @@
+import {altair} from "@lodestar/types";
+import {IBeaconChain} from "../../../chain/index.js";
+
+export async function* onLightclientUpdate(
+  requestBody: altair.LightClientUpdateByRangeRequest,
+  chain: IBeaconChain
+): AsyncIterable<altair.LightClientUpdate[]> {
+  yield await chain.lightClientServer.getUpdates(requestBody.startPeriod, requestBody.count);
```

Comment:
should be wrapped in try/catch?

Reply:
I intentionally left out the try/catch since the spec did not explicitly say `RESOURCE_UNAVAILABLE` should be returned as it did in the other cases. I will look into this again, and if need be, make a PR to the spec.

Reply:
it makes sense to have `RESOURCE_UNAVAILABLE` response, may be the spec just missed it

Reply:
wrapped in try/catch and made a PR to update the spec [here](https://github.com/ethereum/consensus-specs/pull/3003)

Reply:
Alternative is to just return an empty result, or omit the periods that are unavailable, same as for beaconBlocksByRange / beaconBlocksByRoot.